src/cmd/build.js

```diff
@@ -56,8 +92,16 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let extensionName: string = manifestData.name;
+
+  if (manifestData.default_locale) {
```

Comment:
After this if statement, I think we need to add a debugging statement that will show up in `--verbose` mode, like:

    log.debug('Manifest declared default_locale, localizing extension name');

This will help developers understand why they might see errors about a missing a messages file.